rq_scheduler/scheduler.py

```diff
@@ -226,6 +227,55 @@ def change_execution_time(self, job, date_time):
                         raise ValueError('Job not in scheduled jobs queue')
                     continue
 
+    def _rationalize_until(self, until=None):
+        """
+        Rationalizes the `until` argument used by other functions. This function
+        accepts datetime and timedelta instances as well as integers representing
+        epoch values.
+        """
+        if until is None:
+            until = "+inf"
+        elif isinstance(until, datetime):
+            until = to_unix(until)
+        elif isinstance(until, timedelta):
+            until = to_unix((datetime.utcnow() + until))
+        return until
+
+    def get_job_count(self, until=None):
```

Comment:
Agreed - especially since I will remove the `get_job_count_by_queue` method it doesn't make sense to leave it as `get_job_count` any more.